MainActivity.java

```diff
@@ -0,0 +1,70 @@
+package com.example.xiaomi;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+    TextView displayResultPlaceView;

```

Comment:
Init fields in onCreate instead of initing it every time in methods.

Reply:
fixed: https://github.com/NewAzgardec/Calculator/commit/00ed85c16f56cde62db861590f877dd8cc1a6b31